pages/not-supported.tsx

```diff
@@ -0,0 +1,165 @@
+                </button>
+              </div>
+
+              <div>
+                <p>{t("notSupported.body2")}</p>
+              </div>
+              <ul>
+                <li>
+                  <a href={t("notSupported.chrome-link")}>Chrome</a>
+                </li>
+                <li>
+                  <a href={t("notSupported.edge-link")}>Edge</a>
+                </li>
+                <li>
+                  <a href={t("notSupported.mozilla-link")}>Firefox</a>
+                </li>
+                <li>
+                  <a href={t("notSupported.safari-link")}>
+                    Safari {t("notSupported.safari-description")}
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </main>
+      <Footer formData={formInfo} lang={lang} />
+    </>
+  );
+};
+
+const Footer = ({ formData, lang }: { formData: PublicFormRecord; lang: string }) => {
+  const brandImg = formData?.formConfig?.form;
+  const { t } = useTranslation("common");
+  return (
+    <footer className="incompatible-footer">
+      <div className="incompatible-footer-container">
+        <div className="terms-class">
+          <a href={t("footer.terms.link")}>{t("footer.terms.desc")}</a>
+        </div>
+
+        <div className="fip-class">
+          {brandImg?.brand ? (
+            <img
+              alt={lang == "en" ? brandImg?.brand?.logoTitleEn : brandImg?.brand?.logoTitleFr}
+              src={lang == "en" ? brandImg?.brand?.logoEn : brandImg?.brand?.logoFr}
+            />
+          ) : lang == "en" ? (
+            <img src="/img/sig-blk-en.svg" alt={t("fip.text")} />
+          ) : (
+            <img src="/img/sig-blk-fr.svg" alt={t("fip.text")} />
+          )}
+        </div>
+      </div>
+    </footer>
+  );
+};
+
+const DefaultPage = ({ lang }: { lang: string }) => {
+  return (
+    <>
+      {lang == "en" ? (
+        <img src="/img/sig-blk-en.svg" alt="GoC" />
+      ) : (
+        <img src="/img/sig-blk-fr.svg" alt="GoC" />
+      )}
+    </>
+  );
+};
+
+const BrandPage = ({ formData, lang }: { formData: PublicFormRecord; lang: string }) => {
+  const brand = formData.formConfig.form.brand;
+  return (
+    <>
+      {lang == "en" ? (
+        <img src={brand?.logoEn} alt={brand?.logoTitleEn} />
+      ) : (
+        <img src={brand?.logoFr} alt={brand?.logoTitleFr} />
+      )}
+    </>
+  );
+};
+
+NotSupported.getLayout = function (page: ReactElement) {
+  return <>{page}</>;
+};
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  const con = (await context.req.headers) || "not";
+  let localeProps = {};
+  let formConfig;
+  let split;
+  if (context.locale) {
+    localeProps = await serverSideTranslations(context.locale, ["common", "not-supported"]);
+  }
+  if (con.referer) {
+    const url = new URL(con.referer);
+    split = url.pathname.split("/");
+    formConfig = await getFormByID(parseInt(split[split.length - 1]));
```

Comment:
I feel like there should be some sort of check on this, to ensure that the URL has enough path components to have a form ID.